script/run-unit-test.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 # No test specified; run all of them
 if [[ -z "$1" ]] || [[ $1 == -* ]]; then
-  BABEL_ENV=test $MOCHA test/helper.js --recursive '$SCRIPT_DIR/../test/**/*.unit.spec.js?(x)' "$@"
+    BABEL_ENV=test $MOCHA --recursive '$SCRIPT_DIR/../test/**/*.unit.spec.js?(x)' test/helper.js "$@"
```

Comment:
What's this change for?

Reply:
File need to be loaded in the same order as they are in jenkins so when choma shuffles the files with the seed that failed on Jenkins the tests will actually get run in that order.